src/constants.lua

```diff
@@ -11,6 +11,10 @@ constants.oneYearMs = 31536000 * 1000
 
 -- EPOCHS
 constants.defaultEpochDurationMs = constants.oneDayMs
+constants.decayStartingRewardRate = 0.001
```

Comment:
```suggestion
constants.maxiumRewardRate = 0.001
constants.minimumRewardRate = 0.0005
```
and then use these when computing the current decay rate